src/components/Base/index.js

```diff
@@ -16,3 +16,4 @@ export Timeline from './Timeline';
 export Tooltip from './Tooltip';
 export Image from './Image';
 export Upload from './Upload';
+export Link from './Link';
```

Comment:
Mybe rename a new name, Link is a compoent  of React .

Reply:
For the Link to doc website